### DISCUSSION TITLE
Resize direct message popup. Click-drag?

### DISCUSSION THREAD

**Discussion by kaymarierose** [AprilSylph/XKit-Rewritten#1105] [Suggestion Box]
So, basically, the tiny window that pops up for DMs makes it super annoying to read longer messages. It's so small, and it can't move anywhere, you scroll forever. If possible, could I possibly suggest the ability to make it larger? Either by some preset sizes, or the ability to click and drag the window size like you would any windows application. Even if it has set boundaries or size limits. I don't know if that's something xkit is capable of messing with, but it would be a huge QOL improvement. Thank you for your consideration, and the work you've put into xkit so far.

**Comment by marcustyphoon** [comment]
I haven't gone and looked for it, but I believe Staff mentioned in one of the posts on the WIP or Changes blogs that they were working on a redesign of the messaging interface! If that's true, and hopefully it is, then we'd of course hope that it's improved. Message https://www.tumblr.com/support with your feedback!

If it doesn't wind up getting changed, I'd be interested in trying to do this in XKit, certainly.
